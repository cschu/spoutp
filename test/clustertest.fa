>seq1
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq2
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq3
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq4
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq5
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq6
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq7
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq8
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq10
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq11
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq12
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq13
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq14
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq15
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq16
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq17
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCT
CTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGT
GAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTA
TCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACT
TTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGAC
CAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAAC
GACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seqX
AAGCACAGATATTAGACAGTAGGACCAGATTAGACAGTAGAC
