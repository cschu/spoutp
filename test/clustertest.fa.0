>seq1
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq2
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq3
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq4
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq5
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq6
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
