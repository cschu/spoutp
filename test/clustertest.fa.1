>seq7
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq8
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq10
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq11
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq12
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq13
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
