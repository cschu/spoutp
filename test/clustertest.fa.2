>seq7
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq8
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
>seq10
ATGGTGCACGCAACCTCCCCGCTGCTGCTGCTGCTGCTGCTCAGCCTGGCTCTGGTGGCTCCCAGCCTCTCTGCCAGAAAGTGCTCGCTGACTGGGGAATGGGACAACGACCTGGGCTCCATCATGACCATCGGAGCTGTGAACGACAATGGCGAGTTCGATGGCACCTACATCACAGCTGTAGCAGATAATCCAGGAAACATCACGCTATCACCACTGCTTGGGATCCAACACAAAAGAGCCAGCCAGCCCACTTTTGGCTTCACTGTCCATTGGAACTTTTCAGAGTCCACCAGTGTCTTTGTGGGCCAGTGCTTCGTGGACAGGAGCGGAAAGGAGGTCCTGAAGACCAAATGGCTGCAACGGTTAGCAGTTGATGACATTAGTGATGACTGGATAGCTACCAGGGTCGGCAACAACGACTTCACTCGCCAGCACACAGTGGAAGAGTGA
