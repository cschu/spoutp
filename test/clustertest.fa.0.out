MVHATSPLLLLLLLSLALVAPSLSARKCSLTGEWDNDLGSIMTIGAVNDNGEFDGTYITAVADNPGNITLSPLLGIQHKRASQPTFGFTVHWNFSESTSVFVGQCFVDRSGKEVLKTKWLQRLAVDDISDDWIATRVGNNDFTRQHTVEE*
MVHATSPLLLLLLLSLALVAPSLSARKCSLTGEWDNDLGSIMTIGAVNDNGEFDGTYITAVADNPGNITLSPLLGIQHKRASQPTFGFTVHWNFSESTSVFVGQCFVDRSGKEVLKTKWLQRLAVDDISDDWIATRVGNNDFTRQHTVEE*
MVHATSPLLLLLLLSLALVAPSLSARKCSLTGEWDNDLGSIMTIGAVNDNGEFDGTYITAVADNPGNITLSPLLGIQHKRASQPTFGFTVHWNFSESTSVFVGQCFVDRSGKEVLKTKWLQRLAVDDISDDWIATRVGNNDFTRQHTVEE*
MVHATSPLLLLLLLSLALVAPSLSARKCSLTGEWDNDLGSIMTIGAVNDNGEFDGTYITAVADNPGNITLSPLLGIQHKRASQPTFGFTVHWNFSESTSVFVGQCFVDRSGKEVLKTKWLQRLAVDDISDDWIATRVGNNDFTRQHTVEE*
MVHATSPLLLLLLLSLALVAPSLSARKCSLTGEWDNDLGSIMTIGAVNDNGEFDGTYITAVADNPGNITLSPLLGIQHKRASQPTFGFTVHWNFSESTSVFVGQCFVDRSGKEVLKTKWLQRLAVDDISDDWIATRVGNNDFTRQHTVEE*
MVHATSPLLLLLLLSLALVAPSLSARKCSLTGEWDNDLGSIMTIGAVNDNGEFDGTYITAVADNPGNITLSPLLGIQHKRASQPTFGFTVHWNFSESTSVFVGQCFVDRSGKEVLKTKWLQRLAVDDISDDWIATRVGNNDFTRQHTVEE*
